perfplot/main.py

```diff
@@ -19,6 +33,7 @@ def __init__(
         logx,
         logy,
         automatic_order,
+        time_unit=None,
```

Comment:
Last nitpick: Let's call this `"auto"`. After that change, it's good to go.

Reply:
Done! Updated it in the README as well and then realized the tests need to change too after they failed...what are tests for right? 🤣